src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Subscriptions/MSBuildDependencies/AnalyzerDependencyFactory.cs

```diff
@@ -3,7 +3,7 @@
 namespace Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Subscriptions.MSBuildDependencies;
 
 [Export(typeof(IMSBuildDependencyFactory))]
-[AppliesTo(ProjectCapability.DependenciesTree)]
+[AppliesTo(ProjectCapability.DependenciesTree + " & " + ProjectCapability.DotNet)] // There's no "AnalyzerReferences" capability
 internal sealed class AnalyzerDependencyFactory : MSBuildDependencyFactoryBase
```

Comment:
Are analyzers applicable to F#? If not you may need to check for VB or C# instead of DotNet.

Reply:
Good point! It seems that F# does not have analyzers, at least in the same sense as Roslyn. I'll update this.